lib/watch/index.js

```diff
@@ -75,6 +75,7 @@ const watch = function (args) {
           console.log(err)
           stop(watcher, err)
         }
+        childs.push(run('restart'))
```

Comment:
I think this whole try-catch needs a fix. The condition `if (!childs.length === 0)`  can never be `true`, because `!childs.length`  will either be `true`  or `false`, but never `0`.

Reply:
I think it actually meant length !== 0 which I changed in #107